project/plugins.sbt

```diff
@@ -13,3 +13,5 @@ addSbtPlugin("org.scalameta" % "sbt-scalafmt"    % "2.4.6")
 addSbtPlugin("com.typesafe"  % "sbt-mima-plugin" % "1.1.0")
 
 addSbtPlugin("com.dwijnand" % "sbt-dynver" % "4.1.1")
+
+addDependencyTreePlugin
```

Comment:
Did you use any task besides `dependencyTree`? `dependencyTree` comes by default with sbt, no need to add `addDependencyTreePlugin`.

Reply:
@mkurz This is already removed, thanks.